### DISCUSSION TITLE
🎉Welcome to "you-get.exe" Discussions!

### DISCUSSION THREAD

**Discussion by LussacZheng** [LussacZheng/you-get.exe#1] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

* ❓ Ask questions you’re wondering about.
* 💡 Share ideas.
* 💬 Engage with other community members.
* 😄 Welcome others and are open-minded. Remember that this is a community we build together.

To get started, comment below with an introduction of yourself and tell us about what you do with this community, or open a [new discussion](https://github.com/LussacZheng/you-get.exe/discussions/new).

**Comment by demonluo** [comment]
can you do the same for ykdl? it also doesn't release executable for Windows since ykdl v1.7.0 which is as annoying as you-get w/o the .exe file...
https://github.com/SeaHOH/ykdl

**Reply by LussacZheng** [reply]
The [ykdl Wiki](https://github.com/SeaHOH/ykdl/wiki/Windows-Build-script) describes how to build executable file for Windows.

And I haphazardly/simply bundled an executable file of ykdl. You can download and test it: [ykdl.zip](https://github.com/LussacZheng/you-get.exe/files/7497447/ykdl.zip)

**Comment by demonluo** [comment]
cool thanks...

**Reply by LussacZheng** [reply]
Try this instead: [ykdl.build.2.zip](https://github.com/LussacZheng/you-get.exe/files/7498199/ykdl.build.2.zip) , which is built with `Python 3.9.8` , and complements some missing dependencies compared to the previous one.

**Comment by demonluo** [comment]
ok, thanks...
